fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -207,6 +207,8 @@ public enum ErrorCode {
             "Unknown resource group '%s' "),
     ERR_BAD_CATALOG_AND_DB_ERROR(5080, new byte[] {'4', '2', '0', '0', '0'},
             "Unknown catalog.db '%s'"),
+    ERR_CATALOG_EXISTED_ERROR(5607, new byte[] {'4', '2', '0', '0', '0'},
```

Comment:
we just defined 5700-5799 as the external range. please use 5700 in the patch

Reply:
Got it. Thank.

Reply:
Done @stephen-shelby